src/data/atlas-search-index.ts

```diff
@@ -56,6 +56,12 @@ export const SearchIndex: IndexMappings = {
         searchAnalyzer: 'lucene.english',
         type: 'string',
       },
+      strippedSlug: {
+        analyzer: 'lucene.keyword',
+        searchAnalyzer: 'lucene.keyword',
+        store: true,
```

Comment:
what is this `store` option?  [atlas search docs ](https://www.mongodb.com/docs/atlas/atlas-search/stored-source-definition/)reads `doesn't index stored fields and so you can't query these fields, but you can retrieve stored fields at query time using the [returnStoredSource](https://www.mongodb.com/docs/atlas/atlas-search/return-stored-source/#std-label-fts-return-stored-source-option) option`

Reply:
I don't actually remember why I put this here, so I'm removing and testing without the option 🤔